### ISSUE TITLE
1.7-a.230829 : option to add a red margin

### ISSUE THREAD

**Issue by SeeDborg** [OpenBoard-org/OpenBoard#803]
I'm glad we can now use the "seyes" sheets in Openboard 1.7
But in France we have squares and a red margin too :

![image](https://github.com/OpenBoard-org/OpenBoard/assets/7523821/720270b8-cb9f-4954-9440-35f9bc72a6df)

Would it be possible to add an option to add that ?

**Comment by sebojolais** [comment]
In next 1.7, with the parameter SeyesRuledBackground=true in the file OpenBoard.config and if you selected the Seyes background, you got something like your image. 
There is already the red margin. So there is no option, the red margin is by default.
I am sorry, I do not see what you mean by squares.

**Comment by SeeDborg** [comment]
This is perfect.
In the final release, do you plan to add this option (and the option of fullscreen or not) in the parameters ?

Some teachers may have a had time to find the .config and edit it.

**Comment by sebojolais** [comment]
Seyes background is a local feature. Everybody will not benefit of this feature, only french users.
In this case the deal is :
1. no change in the GUI (for example, no option in the preferences window)
2. The feature is disable by default and the behavior could be adjusted by the configuration file.

It was the initial plan #640 and the requirements in order to keep the pull request accepted.

**Comment by SeeDborg** [comment]
I really don't understand why the option can't be present in the parameters and disabled by default.
For many teachers, finding and editing the .config will just be impossible.

What about the Fullscreen option ?

**Comment by sebojolais** [comment]
You know I am also a primary school teacher in France, I agree it is the truth for someones of us.

But keep in mind that you have to set the parameter in the config file only once and after that you got Seyes backgroung by default on every new page.
The config file location is described here :
[https://github.com/OpenBoard-o](https://github.com/OpenBoard-oknow)[rg/OpenBoard/wiki/Settings](https://github.com/OpenBoard-org/OpenBoard/wiki/Settings)
In a positive point of view, it is better than nothing, is not it ?

I can not answer for the fullscreen option because I am just the small contributor of the seyes feature.

**Comment by SeeDborg** [comment]
I agree, it's better than nothing but you have to agree that it's anti-ergonomic to have an option hidden in a .config file.

For you and I it won't be an issue, but many users will simply never use the feature simply because it's not visible and they won't even be aware that it exists...
